library/signalfx/detectors/against_recent/streams.flow

```diff
@@ -21,6 +21,27 @@ def clamper(stream, historical_window_, num_stddev, calculation_mode):
         return max(bot, min(stream, top)) if stream is not None else None
 
 
+def median_iqr_thresholds(stream, current_window=duration('5m'), historical_window=duration('1h'),
+                          fire_num_iqr=2.2, clear_num_iqr=1.85):
+    current_window_ = duration(current_window)
+    historical_window_ = duration(historical_window)
+
+    new_s = stream.timeshift(current_window_)
+
+    iqr = new_s.percentile(75, over=historical_window_) - new_s.percentile(25, over=historical_window_)
+    median = stream.median(over=historical_window_)
```

Comment:
you want the median of `new_s`
also, don't use a method name as a variable name

Reply:
I have changed this. Is it fine now?